pom.xml

```diff
@@ -271,7 +271,7 @@
 					<archive>
 						<manifest>
 							<addClasspath>true</addClasspath>
-							<mainClass>io.smartdatalake.app.DefaultSmartDataLakeBuilder</mainClass>
+							<mainClass>io.smartdatalake.app.LocalSmartDataLakeBuilder</mainClass>
```

Comment:
Does the name "Default" SmartDataLakeBuilder still make sense if it's not the default anymore?

Reply:
If we're going to change it, now would seem to be a good time! What would you recommend? ClusterSDLB? I have no Mesos experience, but think we could use the 'Default' on any cluster manager. Would also fit as a contrast to Local. Interested in your thoughts.

Reply:
...but doesn't cover standalone remote master. Default might not be so bad after all - Local is really the exception, it jist happens to be the usual entry point.

Reply:
I also don't have a better name in mind for now. Let's leave it like that for the moment.